src/dualscreen.tsx

```diff
@@ -41,6 +42,16 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin {
     this._pipPosition = this.config.position;
   }
 
+  getUIComponents() {
```

Comment:
what happens if someone changes media to a none dual screen one? 
What happens if you try to remove the component and it is not there?

Reply:
We didn't handle at all changing media after loading a dual screen. for example, changing to a non dual media will result the secondary previous media still open in the PIP.
Same will go for the settings button removal.

Regarding the second question - Nothing actually happens if you try to remove a component that doesn't exists